.travis.yml

```diff
@@ -1,8 +1,12 @@
 language: node_js
 
 node_js:
-  - 11
   - 10
+  - 11
+  - 12
+  - 13
+
+cache: yarn
```

Comment:
Does it help with speed of build?

Reply:
actually, checked and fresh install is faster (1-2 s). should drop ?

Reply:
removed it and updated PR title